Helper/Order.php

```diff
@@ -727,6 +737,14 @@ public function saveCustomerCreditCard($reference, $storeId)
                 return false;
             }
 
+            $doesCardExist = $this->customerCreditCardCollectionFactory->create()
+                    ->getCreditCards($customerId, $boltConsumerId, $boltCreditCardId)
+                    ->getSize() > 0;
+
+            if ($doesCardExist) {
+                return false;
+            }
+
```

Comment:
Maybe `doesCardExist` should be transformed into a method. Of the Collection class perhaps?